src/dynamicprompts/parser/parse.py

```diff
@@ -227,13 +253,23 @@ def create_parser() -> pp.ParserElement:
     return prompt
 
 
-def parse(prompt: str) -> Command:
+def parse(
+    prompt: str,
+    parser_config: ParserConfig,
```

Comment:
I think the top-level `parse` function should use the default config unless otherwise specified, so downstream users don't need to change things.
```suggestion
    parser_config: ParserConfig = default_parser_config,
```

Reply:
My concern was that if we aren't explicit about requiring it, then one generator might use it and another one doesn't. The idea was that at least person who writes a generator will be forced to consider it.